flink-libraries/flink-gelly/src/main/java/org/apache/flink/graph/library/HITSAlgorithm.java

```diff
@@ -273,4 +289,23 @@ public void sendMessages(Vertex<K, Tuple2<DoubleValue, DoubleValue>> vertex) {
 			return initVertexValue;
 		}
 	}
+
+	public static class AuthorityEdgeMapper<K, EV> implements MapFunction<Edge<K, EV>, String> {
```

Comment:
@greghogan  @vasia In the use of HITS algorithm, bidirectional edges is not very common, so I do not have bidirectional edge determination logic written into the code and all of the edges have joined their reverse edges.